docs/operating-scylla/procedures/backup-restore/backup.rst

```diff
@@ -28,29 +28,35 @@ Therefore, an increasing amount of disk space may be held up by snapshots, hence
 
 **Procedure**
 
-| 1. Data can only be restored from a snapshot of the table schema, where data exists in a backup. Backup your schema with the following command:
+On each node, run:
 
-| ``$: cqlsh -e "DESC SCHEMA" > <schema_name.cql>``
+| ``$ nodetool snapshot``
 
-For example:
-
-| ``$: cqlsh -e "DESC SCHEMA" > db_schema.cql``
-
-|
-| 2. Take a snapshot, including every keyspace you want to backup.
-
-| ``$ nodetool snapshot <KEYSPACE_NAME>``
-
-| For example:
+For example, to snapshot a particular keyspace:
 
 | ``$ nodetool snapshot mykeyspace``
-
-| The snapshot is created under Scylla data directory ``/var/lib/scylla/data``
+|
+| The snapshot is created under the Scylla data directory ``/var/lib/scylla/data``
 | It will have the following structure:
+|
 | ``keyspace_name/table_name-UUID/snapshots/snapshot_name``
-
+|
 | For example:
-| ``/mykeyspace/team_roster-91cd2060f99d11e6a47a000000000000/snapshots/1487847672222``
+|
+| ``mykeyspace/team_roster-91cd2060f99d11e6a47a000000000000/snapshots/1487847672222``
+|
+| By default, nodetool automatically names the snapshot using the current time since the system EPOCH, in milliseconds.
+| For advanced options, like setting the snapshot name (a.k.a. tag) or taking a snapshot of a list of keyspaces, see `nodetool snapshot`_.
+|
+| Note: to restore data in a given snapshot, a corresponding schema must be used.
+| Scylla automatically describes the schema for each table in the snapshot, storing the result in a file named ``schema.cql`` in each snapshot subdirectory.
+| However, it is recommended to snapshot also the system keyspaces when taking a full backup, so the schema can be restored from the system tables.
```

Comment:
It's not that easy to fix, otherwise it might have been fixed already.
Until then, we should recommend the best practices that are correct for the implementation at its current state.